CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # App Center SDK for iOS and macOS Change Log
 
-## Version 1.13.1 (Not yet released)
+## Version 1.14.0
```

Comment:
please use 1.13.1 as its bug fix only.

Reply:
What Guillaume says. Post-Release of 1.13.1, the next version will be 1.13.2.